package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "rsschool-codejam",
+  "version": "1.0.0",
+  "description": "Implementation of some functions in JS",
+  "main": "test.js",
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Rick42Morty/rsschool-codejam.git"
+  },
+  "author": "Yauheni Yarotski",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Rick42Morty/rsschool-codejam/issues"
+  },
+  "homepage": "https://github.com/Rick42Morty/rsschool-codejam#readme",
+  "devDependencies": {
+    "eslint": "^5.8.0",
+    "eslint-config-airbnb-base": "^13.1.0",
+    "mocha": "^5.2.0"
```

Comment:
In order to use _eslint-config-airbnb-base_ should be added one more package _eslint-plugin-import_ :

`"eslint-plugin-import": "2.14.0",`